exonum/src/runtime/error.rs

```diff
@@ -141,22 +196,22 @@ impl Display for ErrorKind {
 /// }
 /// ```
 ///
-/// Deriving errors in runtimes:
+/// Deriving errors for a runtime:
 ///
 /// ```
 /// use exonum_derive::IntoExecutionError;
 ///
-/// // Define runtime specific errors.
+/// /// Runtime-specific errors.
 /// #[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd, IntoExecutionError)]
 /// #[execution_error(kind = "runtime")]
 /// enum SampleRuntimeError {
-///     /// Incorrect information to call the transaction.
-///     IncorrectCallInfo = 1,
-///     /// Incorrect transaction payload.
-///     IncorrectPayload = 2,
+///     /// Incorrect deployment specification format.
+///     MalformedDeploySpec = 1,
+///     /// Could not compile the artifact.
+///     CompilationFail = 2,
 /// }
 /// ```
-#[derive(Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd, Fail)]
+#[derive(Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd, Fail, BinaryValue)]
```

Comment:
Regarding `BinaryValue`: is  `ExecutionError` really meant to be stored in the database? I thought that description is only needed for logging/reporting, and in the database itself we only store the kind.

Reply:
We stores description of errors in database but doesn't use them in the object hash.

Reply:
I understand that, but nevertheless we can (can we?) avoid it to optimize storage usage.